lib/Eclair/LLVM/Vector.hs

```diff
@@ -34,10 +34,10 @@ type Destructor = Operand -> IRCodegen ()
 
 data CGState
   = CGState
-  { externals :: Externals
```

Comment:
Same comment here as in symbol table, revert back to the old names and turn off the specific warning for this module.